planning/behavior_path_planner/include/behavior_path_planner/utils/avoidance/avoidance_module_data.hpp

```diff
@@ -238,6 +238,16 @@ struct AvoidanceParameters
   // avoidance path will be generated unless their lateral margin difference exceeds this value.
   double avoidance_execution_lateral_threshold;
 
+  double quantize_filter_threshold;
+
+  double same_grad_filter_1_threshold;
+
+  double same_grad_filter_2_threshold;
+
+  double same_grad_filter_3_threshold;
+
+  double sharp_shift_filter_threshold;
+
```

Comment:
Since the other parameters in `AvoidanceParameters` has some comments to describe what they are, it is a bit weird when these parameters doesn't have their descriptions.

It's just optional, so there is no issues, at least for me.

Reply:
OK :+1: I'll add brief description.

Reply:
Added in https://github.com/autowarefoundation/autoware.universe/pull/3624/commits/6c2daee7da4fb08060d726757e30f26eb4a9a97b.